base/base/cgroupsv2.cpp

```diff
@@ -71,3 +71,27 @@ fs::path cgroupV2PathOfProcess()
     return {};
 #endif
 }
+
+std::optional<std::string> getCgroupsV2PathContainingFile(std::string_view file_name)
```

Comment:
The function name could be more clear, maybe `getCgroupsV2BottommostPathContainingFile`?